inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/resources/docs/.vuepress/config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Another hidden file?

Reply:
According to VuePress Configuration page this seems also correct.
I don't see alternatives exception other file formats (`yml` or `toml`).
However, as this documentation for pdfanno does not contain much and also some wrong information, I could remove the complete `docs` folder.

Reply:
Removing the docs folder sounds sensible.